### ISSUE TITLE
English version for archival

### ISSUE THREAD

**Issue by heylix** [vkazanov/everything-about-cron-post#1]
Hi, I'd like to cite your post in an article, but it can't be archived on medium.com because medium.com doesn't like archival of their content on archive.org and you get a weird Javascript Error 500:

https://web.archive.org/web/20200926175146/https://medium.com/bumble-tech/cron-in-linux-history-use-and-structure-70d938569b40

Archive.is doesn't seem to be affected by the medium.com bumblefuckery:

https://archive.is/BU3m2

… but the owner of archive.is is a bit weird sometimes. "Feeling archived, might delete later" basically.

Can you please upload the English version here? It's a pretty good deep dive into cron, I really enjoyed reading it! :)

**Comment by vkazanov** [comment]
Yes, sure. :-) Don't remember why I didn't upload a draft. I think this might require some editing but I should find an hour or two sometime this week.

**Comment by vkazanov** [comment]
@heylix I've added the draft of the post in English
